CHANGELOG.md

```diff
@@ -32,6 +33,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `CategoricalParameter` and `TaskParameter` no longer incorrectly coerce a single
   string input to categories/tasks
 - `farthest_point_sampling` no longer depends on the provided point order
+- `RandomForestSurrogate` now produces proper batch predictions
```

Comment:
`Fixing batch recommendations for RFS` sounds a bit nicer :D

Reply:
Can do, but please provide a version that
* uses the same language as in the other points (i.e. we never start with a verb like "fixing")
* uses `prediction` instead of `recommendation` (the latter concept does not exist on the surrogate level)

Reply:
How about simply having "Batch predictions for `RandomForrestSurrogate` "? Can however also stick with the previous formulation of you prefer that.

Reply:
changed 👍🏼